<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProjectDialog</class>
 <widget class="QDialog" name="NewProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Project</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Project Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="project_name">
      <property name="statusTip">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Resolution:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="resolution_x">
      <property name="suffix">
       <string>px</string>
      </property>
      <property name="prefix">
       <string>X:</string>
      </property>
      <property name="maximum">
       <number>999999</number>
      </property>
      <property name="value">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="resolution_y">
      <property name="suffix">
       <string>px</string>
      </property>
      <property name="prefix">
       <string>Y:</string>
      </property>
      <property name="maximum">
       <number>99999999</number>
      </property>
      <property name="value">
       <number>720</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="ok_button">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel_button">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Project Path:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="project_path">
      <property name="statusTip">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="file_browser_button">
      <property name="text">
       <string>Browser</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ok_button</sender>
   <signal>clicked()</signal>
   <receiver>NewProjectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel_button</sender>
   <signal>clicked()</signal>
   <receiver>NewProjectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>project_name</sender>
   <signal>textChanged(QString)</signal>
   <receiver>NewProjectDialog</receiver>
   <slot>projectPathChange(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>project_path</sender>
   <signal>textChanged(QString)</signal>
   <receiver>NewProjectDialog</receiver>
   <slot>projectPathChange(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>file_browser_button</sender>
   <signal>clicked()</signal>
   <receiver>NewProjectDialog</receiver>
   <slot>openFilePath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>projectPathChange(QString)</slot>
  <slot>openFilePath()</slot>
 </slots>
</ui>
